docs/rfcs/033-merged-compute-image.md

```diff
@@ -0,0 +1,54 @@
+## Merged compute image
+https://github.com/neondatabase/neon/issues/6685
+
+### Motivation:
+It's hard to manage compute pools for 3 Postgres versions.  
```

Comment:
Do we have a plan or an idea of what we'll do with 4-5-6 supported Postgres versions? Adding them to the image will increase its size, but that may be fine and not an issue for us.